### ISSUE TITLE
unable to use the '--force-exclusion' flag in the rubocop executable path

### ISSUE THREAD

**Issue by bartocc** [AtomLinter/linter-rubocop#43]
I get the following error

Uncaught Error: ENOENT: no such file or directory, stat '~/.rvm/gems/ruby-2.2.2/bin/rubocop --force-exclusion'

**Atom Version**: 0.207.0
**System**: Mac OS X 10.10.3
**Thrown From**: [linter](https://github.com/AtomLinter/Linter) package, v0.12.6

**Comment by Fustrate** [comment]
It's actually already there. I edited the main Linter package to see the log, and here's the command it runs (without anything but the executable in the path):

`/Users/steven/.rbenv/versions/2.2.2/bin/rubocop --force-exclusion --format emacs --config /Users/steven/Development/MyProject/.rubocop.yml /var/folders/5m/fw5tzfv51pn_182x0_1js9v80000gn/T/AtomLinter115529-76624-1w4doni/schema.rb`

As far as I can tell, the exclusion problem (which is, I assume, why you're trying to add `--force-exclusion`) stems from using a *completely* different path to the file from the one Rubocop would exclude - and that's how the Linter package works. As far as Rubocop is concerned, it's just a file somewhere else on the system that has nothing to do with the files in the include/exclude paths, even if they share the same name. There's no `.rubocop.yml` in `/var/folders/5m/fw5tzfv51pn_182x0_1js9v80000gn/T/AtomLinter115529-76624-1w4doni`, and it doesn't even preserve the `/db/` part of the path, so it's all sorts of wacky.

I'm not sure if this has to be "fixed" in rubocop, or the linter should parse the found `.rubocop.yml` file and  not try to lint excluded files in the first place.

**My findings can be tested** by running these two commands from the project root (assuming it's a Rails app - replace `db/schema.rb` with any excluded file if not):

```shell
cp --parents db/schema.rb ~/Desktop
rubocop ~/Desktop/db/schema.rb --force-exclude --config .rubocop.yml
```

**Comment by renchap** [comment]
:+1: this is a very annoying issues to not have the rubocop exclude rules applied inside atom, rendering the rubocop linter quite painful to use.

**Comment by tony612** [comment]
:+1:

**Comment by bartocc** [comment]
Guys, can you review the PR #75 ?

To test it :
- remove the official linter-rubocop package
- clone [my 'understand-exclude-parameters' branch](https://github.com/bartocc/linter-rubocop/tree/understand-exclude-parameters)
- cd to the cloned repo
- run `apm link`
- add '--force-exclusion' to the Additional Arguments in the package's config
- append/merge this in your .rubocop.yml config file

```yml
AllCops:
  Exclude:
    - '**/db/schema.rb'
```

- open a "schema.rb" file in Atom, lint and see if you get errors or not

**Comment by caseywebdev** [comment]
This is fixed in `0.4.0`
